srcpkgs/jalv/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'jalv'
+pkgname=jalv
+version=1.6.6
+revision=1
+build_style=waf3
+configure_args="$(vopt_if portaudio '--portaudio') --no-gtkmm
+ $(vopt_if gtk '' '--no-gtk2') $(vopt_if gtk3 '' '--no-gtk3')
```

Comment:
`gtk2` shouldn't be made available at all.

Reply:
okay I removed it

Reply:
oops okay now I fully removed it